operations/README.md

```diff
@@ -82,6 +82,15 @@ This section describes all operations supported by the API, organised here by th
 | [Add alternative payment](payments.md#add-alternative-payment) | Adds a new alternative payment to a specified customer |
 | [Add outlet bills](outletbills.md#add-outlet-bills) | Adds new outlet bills with their items |
 
+## Messages
+
+| <div style="width:200px">Operation or Endpoint</div> | Description |
+| :-- | :-- |
+| [Get all message threads](operations/messagethreads.md#get-all-message-threads) | Returns all message threads created by the client application |
```

Comment:
This is great, but I don't think the links will work? e.g. 'operations/messagethreads.md' should just be 'messagethreads.md' because we are already in the 'operations' directory.